src/gt4py/eve/extended_typing.py

```diff
@@ -77,10 +73,6 @@
         Set as AbstractSet,
         ValuesView as ValuesView,
     )
-    from contextlib import (
-        AbstractAsyncContextManager as AsyncContextManager,
-        AbstractContextManager as ContextManager,
-    )
```

Comment:
Why are those removed? IIRC those imports are here just to mimic the deprecated aliases in `typing` for older python versions. Once we drop support for 3.8 and 3.9 (hopefully in a matter of weeks) we should delete all these aliases, but until then I think we should keep them for consistency.

Reply:
I removed them because the vscode language server for python marked them as unused (unlike others in that category). Since 3.8 and 3.9 versions of tests still pass, they might actually be unused. However, I didn't see that they are part of a code block guarded for 3.9 only. If the plan is to drop support for 3.8 and 3.9 in the next weeks, I agree, that there's no need to change anything here since we'll just get rid of the whole block with that PR. I'll revert my changes here :+1: